python/semantic_kernel/memory/memory_store_base.py

```diff
@@ -1,12 +1,120 @@
 # Copyright (c) Microsoft. All rights reserved.
 
-from abc import ABC
+from abc import abstractmethod
+from typing import List, Tuple
 
-from semantic_kernel.connectors.ai.embeddings.embedding_index_base import (
-    EmbeddingIndexBase,
-)
-from semantic_kernel.memory.storage.data_store_base import DataStoreBase
+from numpy import ndarray
 
+from semantic_kernel.memory.memory_record import MemoryRecord
 
-class MemoryStoreBase(DataStoreBase, EmbeddingIndexBase, ABC):
-    pass
+
+class MemoryStoreBase:
+    @abstractmethod
+    async def create_collection_async(
+        self,
+        collection_name: str
+        # TODO: cancel token
```

Comment:
Do we need all these TODOs? If so, it's unclear to me what someone would need to do to address it

Reply:
removed the todos and created a task to track the work